tests/installation/data_integrity.pm

```diff
@@ -16,7 +16,10 @@ use testapi;
 use data_integrity_utils 'verify_checksum';
 
 sub run {
-    verify_checksum;
+    my $dir_path;
```

Comment:
Without an orphan declaration (but it is again a matter of taste :) ). Feel free to leave it as it is.
```
$dir_path = check_var('BACKEND', 'svirt') ? '/var/lib/libvirt/images/'
           : check_var('VIRSH_VMM_FAMILY', 'hyperv') ? get_var('HYPERV_DISK', 'D:') . '\\cache\\' : undef;
```

Reply:
A ternary operator will make the code more difficult to read, just a comment would do.